ethereum/contracts/SwapCreator.sol

```diff
@@ -160,11 +174,11 @@ contract SwapCreator is ERC2771Context, Secp256k1 {
     }
 
     // Bob can claim if:
-    // - Alice has set the swap to `ready` or it's past t_0 but before t_1
+    // - (Alice has set the swap to `ready` or it's past timeout0) and it's before timeout1
     function claim(Swap memory _swap, bytes32 _s) public {
         _claim(_swap, _s);
 
-        // send eth to caller (Bob)
+        // send ether to swap claimant
```

Comment:
"Claimer" already is a noun. We probably shouldn't introduce a new noun. "Claimant" is usually only used in legal contexts.

I've actually been standardizing on ETH (all caps) in the Go code. When looking at modern documentation, it seems like ETH is used more often. Not that there is anything wrong with "ether". It is not why I wrote this comment.